### PR TITLE
Sum of digits implemented in JS. Closes #1610

### LINKED ISSUE #1610: Sum of Digits in JS.
I would like to implement this problem in JavaScript. (GSSoc20)

### PR DESCRIPTION
**Pull request by JanaSabuj**
Fixes #1610 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Implemented the said problem in JS.

#### Languages Used:
- JavaScript

#### Files Added:
- Sum_Of_Digits.js

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by JanaSabuj** [comment]
@singh-shreya6 Please review it.

File: Sum_Of_Digits/Sum_Of_Digits.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Output:
+The sum of digits of 12345 is 15
+
+Input:
+1000
+
+Output:
+The sum of digits of 1000 is 1
+*/
```
[the remark this answers is not in the document]
Comment (JanaSabuj) [reply Sum_Of_Digits/Sum_Of_Digits.js]: @singh-shreya6 added a new line. please review it

File: Sum_Of_Digits/Sum_Of_Digits.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/* The sum of digits of a number is the mathematical addition
+of the digits extracted from the number.
+Example: number = 12345, sum = 1 + 2 + 3 + 4 + 5 = 15
+number = 1005, sum = 1 + 0 + 0 + 5 = 6 */
+
+const sumOfDigits = (n) => {
+	let sum = 0 // initialise sum variable to 0
+	while(n > 0){
+		const b = n % 10
```
Comment (singh-shreya6) [inline Sum_Of_Digits/Sum_Of_Digits.js]: It looks like 8 space indentation is there. Please maintain 4 space indentation at all places. The code logic looks fine.

Comment (JanaSabuj) [reply Sum_Of_Digits/Sum_Of_Digits.js]: @singh-shreya6 done. please do review it. sorry for the wrong indentation.

**Comment by JanaSabuj** [comment]
@singh-shreya6 please review it. 
Screenshot from 2020-03-03 09-47-10

**Comment by SakshiShreya** [comment]
Please squash the commits.

**Comment by JanaSabuj** [comment]
@SakshiShreya how to squash the commits? i cannot rebase. it is showing error

**Comment by JanaSabuj** [comment]
@SakshiShreya @asha15 I created a new PR with 1 commit. 
https://github.com/jainaman224/Algo_Ds_Notes/pull/2035

**Comment by JanaSabuj** [comment]
@SakshiShreya @asha15 @jainaman224 @singh-shreya6 I finally squashed the commits. Please review the PR. Ignore the other PR.

**Pull request merged**